pkg/storage/stores/indexshipper/compactor/retention/retention_test.go

```diff
@@ -523,6 +523,13 @@ func TestChunkRewriter(t *testing.T) {
 
 				for _, interval := range expectedChunks[i] {
 					for curr := interval.Start; curr <= interval.End; curr = curr.Add(time.Minute) {
+						// // Test ready to pass/fail when we change the default chunk and head format.
```

Comment:
Should this commented out test already pass? I thought there are no "default" chunk and head formats any more, since they are "encoded" into the schema version.

Reply:
Done.